moses/moses/main/populate_atomspace.h

```diff
@@ -36,12 +40,20 @@ namespace opencog { namespace moses {
  *
  * @return                 LinkValue pointer containing the logical_and.
  */
-void populate(AtomSpace* as, const ITable &itable)
-{
-	OC_ASSERT(false, "populate_atomspace Not implemented yet!");
-}
+void populate(AtomSpace *as,  ITable &itable);
+
+/**
+ * Populate the given atomspace using Otable
+ * @param Atomspace&
+ * @param OTable&
+ * @return
+ */
 
-}
-}
+
+ValuePtr vertex_seq_to_value(const vertex_seq& vseq, id::type_node& tnode);
 
 #endif
+} // ~namespace moses
+} // ~namespace opencog
```

Comment:
`#endif` should occur after closing the namespace. Putting it here may cause compiling errors if that file is being included multiple times.